Examples/RKSearchExample/RKSearchExample.xcodeproj/project.pbxproj

```diff
@@ -338,7 +338,7 @@
 				GCC_WARN_ABOUT_RETURN_TYPE = YES;
 				GCC_WARN_UNINITIALIZED_AUTOS = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 6.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 8.1;
```

Comment:
You should endeavour to always use the highest point version of the base deployment target, otherwise you open yourself up to loads of different bugs. Very few people will be running 8.0 when they could be running 8.4.1